### ISSUE TITLE
Support for Vivaldi?

### ISSUE THREAD

**Issue by lilyseki** [rilian-la-te/vala-panel-appmenu#111]
Not sure if it's possible, as, iirc, Vivaldi is closed source, but being based mostly on Chromium, surely there must be a way?

Thanks!

**Comment by rilian-la-te** [comment]
AFAIK, chromium hints works for Vivaldi also. Can you try it and post screenshots here?

**Comment by rilian-la-te** [comment]
Tried chromium hints myself, not working. Seen a request on Vivaldi forum.

**Comment by lilyseki** [comment]
It works great with Chrome for me. I searched on the Vivaldi forums, but couldn't find anything, though. Did you mean I should post a request?

**Comment by rilian-la-te** [comment]
No, you can look a request by keywords "Unity menu". And bump it:)

**Comment by lilyseki** [comment]
Well, I found numerous threads requesting support for Unity's menu and KDE's Global menu, ranging from two years old, to two weeks old, with very little developer response. So I guess I shouldn't hold my breath!

Thanks, though. ♥
